spec/support/database_cleaner.rb

```diff
@@ -1,23 +0,0 @@
-# frozen_string_literal: true
```

Comment:
This file exists because `database_cleaner` is setup for an old environment. You can ignore it if you don't have this file